### PR TITLE
Orog fix file reorg and fix directory variable standardization

### LINKED ISSUE #1981: Update fix files for HR3/EP5
### Target fix directory

orog

### Type of change

- [ ] Update existing files
- [ ] Remove files
- [ ] Add new files to existing directory
- [X] Add new fix directory

### Any workflow changes needed?

Fix version only

### Related issues

#1730 
#1692 
https://github.com/ufs-community/UFS_UTILS/issues/867

### Pickup location

/scratch2/NCEPDEV/stmp1/Sanath.Kumar/HR3_EP5

### Size

0

### LINKED ISSUE #1984: Update global lake data
### Target fix directory

raw

### Type of change

- [X] Update existing files
- [X] Remove files
- [X] Add new files to existing directory
- [ ] Add new fix directory

### Any workflow changes needed?

No change needed

### Related issues

https://github.com/ufs-community/UFS_UTILS/issues/854

### Pickup location

Hera: /home/Shan.Sun/tmp_lake_txt

### Size

No difference.

### LINKED ISSUE #2175: Standardize fix directory variables
### What new functionality do you need?

Consistency and correctness with the various variables that define fix file subdirectories.

Example problems: `OROFIX` should be `FIX_orog` and `FIXgsm` should be `FIXam`

### What are the requirements for the new functionality?

Standard naming convention for fix directory variables.

### Acceptance Criteria

Fix directory variables are updated, standardized and tests work with updates.

### Suggest a solution (optional)

_No response_

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR makes updates to use the newly reorganized `orog/20231027` fix file version and standardizes the fix file directory variables.

Changes:

1. Update UFS_UTILS hash to https://github.com/ufs-community/UFS_UTILS/commit/ce385cedfa9abd46b0905e8d6486b0339a9e4267
2. Update fix file version file to use new orog timestamp: `orog_ver=20231027`
3. Add ".mx$(OCNRES)" to `oro_data` filenames in scripts
4. Remove ".mx$(ocn_res)_frac" from the directory names for atmosphere resolutions under the `/orog` folder
5. Adds fix file directory variables to `config.base`, many of which are pulled up out of lower scripts; introduce `FIXcpl`
6. Renamed/replaced the following fix file directory variables:

- `FV3JEDI_FIX` --> `FIXgdas`
- `FIXfv3`, `FIX_orog` and `OROFIX` --> `FIXorog`
- `FIX_ugwd` and `OROFIX_ugwd` --> `FIXugwd`
- `FIX_am`, `FIXgsm`, `FIXSYND` --> `FIXam`
- `FIX_lut` --> `FIXlut`
- `FIX_aer` --> `FIXaer`
- `FIX_SFC` --> `FIXsfc`
- `FIX_DIR` --> `FIXgfs`
- `RTMFIX` and `FIXCRTM` --> `CRTM_FIX` (is `CRTM_FIX` in crtm module, so not using `FIXcrtm`)

7. Correct the defaults for `FIXwave` in wave JJOB scripts
8. Update variable names in `*fixed_files.yaml` files, as well as update naming convention for `oro_data` files

Resolves #1984
Resolves #1981
Resolves #2175

# Type of change

- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

- Clone, build, link on WCOSS2
- 1.5 cycle atmos-only low res cycled test on WCOSS2

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Comment by KateFriedman-NOAA** [comment]
Am awaiting conclusion of https://github.com/ufs-community/UFS_UTILS/pull/881 so I can include a new UFS_UTILS hash in this PR before merging after approvals.

File: scripts/exgfs_wave_nawips.sh
Comment (aerorahul) [inline scripts/exgfs_wave_nawips.sh:153]: Interesting way to find out if a file exists and fail if it isn't

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Conditionally approved pending successful completion of CI tests.

File: parm/ufs/fix/gfs/land.fixed_files.yaml
Comment (barlage) [inline parm/ufs/fix/gfs/land.fixed_files.yaml:32]: There should also be a section for soil_color above this.

Comment (aerorahul) [reply parm/ufs/fix/gfs/land.fixed_files.yaml:32]: @barlage 
Thanks for the review.  This part of the code in `parm/ufs/fix/gfs/*yaml` are not exercised at this time, so we are safe.

**Comment by KateFriedman-NOAA** [comment]
The UFS_UTILS hash has been updated to https://github.com/ufs-community/UFS_UTILS/commit/ce385cedfa9abd46b0905e8d6486b0339a9e4267, which includes companion updates for this work. Thanks @GeorgeGayno-NOAA !

**Comment by KateFriedman-NOAA** [comment]
Successfully retested clone, build, link on WCOSS2 after updating UFS_UTILS hash.

**Pull request merged**
